lib/src/ffmpeg/filters/concat_filter.dart

```diff
@@ -11,8 +11,13 @@ class ConcatFilter implements Filter {
         assert(outputVideoStreamCount >= 0),
         assert(outputAudioStreamCount >= 0);
 
+  /// Number of segments (defaults to 2)
```

Comment:
I don't see a default

Reply:
The segment count can now be omitted, so it will reflect the ffmpeg default of 2 as described [here](https://ffmpeg.org/ffmpeg-filters.html#concat). <br>
(When writing this, I changed the n and t attributes inside of the overlay_filter.dart file to mirror the way that concat names the attributes - with them not being the single letters like n).